### DISCUSSION TITLE
I have some notes including

### DISCUSSION THREAD

**Discussion by moalim** [woocommerce/woocommerce#49156] [Checkout Flow]
The logout page is very beautiful. However, I have some notes including:

- It is not possible to hide certain fields when selling digital products, such as shipping options and buyer addresses. Unfortunately, they are mandatory and not optional.

**Comment by nielslange** [comment]
Hello @moalim 👋

Currently, it's not possible to hide the buyers' address, as that information is relevant for tax purposes. For example, when selling digital goods in the EU, the merchant needs to charge the tax of the shoppers country and transfer the tax to the corresponding country. 

As for the shipping options, when creating the product as a virtual product, no shipping-related sections should appear on the Checkout block:

<img width="3008" alt="Screenshot 2024-07-10 at 13 11 13" src="https://github.com/woocommerce/woocommerce/assets/3323310/f44a933b-842e-49e0-8ff8-6dddd07ebe91">
